src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -405,8 +405,8 @@
   <data name="MultiLinePasteDialog.CloseButtonText" xml:space="preserve">
     <value>Cancel</value>
   </data>
-  <data name="MultiLinePasteDialog.Content" xml:space="preserve">
-    <value>You are about to paste text that contains multiple lines. If you paste this text into your shell, it may result in the unexpected execution of commands. Do you wish to continue?</value>
+  <data name="MultiLineWarningText.Text" xml:space="preserve">
+    <value>You are about to paste text that contains multiple lines. If you paste this text into your shell, it may result in the unexpected execution of commands. Do you wish to continue? Here are your clipboard contents:</value>

```

Comment:
I'd encourage you to make this simpler -- don't change this string, and introduce a medium-side header above the text block that just says "Clipboard Contents (Preview)" or something similar